client-app/src/pages/account/views/AccountUsers.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <loading :active.sync="isLoading" :z-index="5000"></loading>
```

Comment:
we don't need z-index in this component, it was used in orders component because we had to draw loader over modal with details